src/test/java/hudson/plugins/tfs/model/ProjectTest.java

```diff
@@ -59,10 +59,16 @@ public void assertConvertServerChange() throws Exception {
     private UserLookup createMockUserLookup(String accountName, String displayName, String emailAddress) {
         UserLookup userLookup = mock(UserLookup.class);
         User user = mock(User.class);
-        // this portion stolen from User.get()
-        final String id = accountName.replace('\\', '_').replace('/', '_').replace('<','_')
-                .replace('>','_');  // 4 replace() still faster than regex
-        // end stolen portion
+
+        //should put this in a utility/getter function somewhere

```

Comment:
Great idea!  Please make it so.  I recommend a static method on `TfsUserLookup`.